src/content/1.0.0/library/clojure/clojure-overview.md

```diff
@@ -0,0 +1,206 @@
+## Clojure
+
+[![cljdoc badge](https://cljdoc.org/badge/com.fluree/db)](https://cljdoc.org/d/com.fluree/db/CURRENT)
+
+Fluree has a Clojure client.
+
+### A note on terminology
+
+If you are familiar with Datomic, Datahike, Datascript, or Crux (or other Clojure triple-store+ databases) you may wonder at the difference in terminology between those databases and Fluree. Fluree uses [RDF](https://en.wikipedia.org/wiki/Resource_Description_Framework) terminology to describe the relationships between entities, or subjects.
+
+Here's a quick glossary of how your knowledge may map on to Fluree's terminology:
+
+Clojure Triple-Store+ term | Fluree term
+-- | --
+entity | subject
+attribute | predicate
+value | object
+datom | flake
+
+### Setup
+Add to the artifact to your dependencies:
+
+Leiningen -
+
+```all
+[com.fluree/db "1.0.0-rc22"]
+```
+
+deps.edn -
+
+```all
+com.fluree/db {:mvn/version "1.0.0-rc22"}
+```
+
+Require the api and establish a connection to your Fluree server:
```

Comment:
I think we should mention a previous step which is to ensure you are running a Fluree ledger.

i.e.:
Ensure you have a running Fluree Ledger service. Fluree can be downloaded <here>, unzip and run ./fluree-start.sh in the directory, or run from docker with the command <docker command here>. More detailed instructions can be found <here - assuming we have a page somewhere on running Fluree>.